src/Vulkan/Silk.NET.Vulkan/VulkanLibraryNameContainer.cs

```diff
@@ -17,7 +17,7 @@ internal class VulkanLibraryNameContainer : SearchPathContainer
         public override string MacOS => "libMoltenVK.dylib";
 
         /// <inheritdoc />
-        public override string Android => "libvulkan.so.1";
+        public override string Android => "libvulkan.so";
```

Comment:
You'll also need to change this in generator.json. While this file isn't regenerated every run, if ever it's deleted it will pop back up with the wrong name...